bitnami/metallb/templates/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "metallb.fullname" . }}-controller
+  labels: {{- include "metallb.labels" . | nindent 4 }}
+    component: controller
+spec:
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels: {{- include "metallb.matchLabels" . | nindent 6 }}
+      component: controller
+  template:
+    metadata:
+      labels: {{- include "metallb.labels" . | nindent 8 }}
+        component: controller
+      {{- if .Values.controller.podAnnotations }}
+      annotations: {{- toYaml .Values.controller.podAnnotations | nindent 8 }}
+      {{- end }}
+    spec:
+{{- include "metallb.controller.imagePullSecrets" . | nindent 6 }}
+      serviceAccountName: {{ include "metallb.controllerServiceAccountName" . }}
+      terminationGracePeriodSeconds: 0
+      securityContext:
```

Comment:
I would make the `securityContext` configurable instead of setting it directly. There are some kind of cluster like Openshift ones where it should not be provided in order to obtain a random user. You can take this as reference: https://github.com/bitnami/charts/blob/de6dba952b698c6e94b8caa8950e947275d700dc/bitnami/grafana/values.yaml#L266

Reply:
Yes missed it here. Thanks for pointing that out.

Reply:
Fixed that now